index.js

```diff
@@ -9,16 +9,14 @@ const takeUntil = sSrc => src => (start, sink) => {
       sourceTalkback = d;
 
       sSrc(0, (st, sd) => {
-        if (done) return;
```

Comment:
this guard is excessive because we shouldnt receive anything from the source after we unsubscribe from it (when dealing with spec-compliant callbags), so there is no point in validating this here, it just cant happen

at the moment if we remove this guard we might see some nasty unhandled errors in the console while running tests, but that's because callbag-from-promise is broken (im preparing a PR fixing it right now)

Reply:
`callbag-from-promise` fix got released already (ive updated the dev dep here), so disregard my comment above